cmake/helpers/CheckDependentLibrariesAVIF.cmake

```
@@ -1,2 +1,23 @@
 define_find_package2(AVIF avif/avif.h avif PKGCONFIG_NAME libavif)
 gdal_check_package(AVIF "AVIF" CAN_DISABLE)
+
+# Check if libavif supports opaque properties
+include(CheckCXXSourceCompiles)
+cmake_push_check_state(RESET)
+set(CMAKE_REQUIRED_INCLUDES ${AVIF_INCLUDE_DIR})
```

Would this work?
```suggestion
set(CMAKE_REQUIRED_INCLUDES "${AVIF_INCLUDE_DIRS}")
```
(Usually singular variables are meant for Find modules input, but not for output. But real world has all kind of patterns. And then there is AVIF::AVIF with its properties, as used below.)

---

confirmed to work on my side